src/coverage-system/sectionfinder.ts

```diff
@@ -33,10 +32,13 @@ export class SectionFinder {
         sections: Map<string, Section>,
     ): Section | undefined {
         const sectionsArray = Array.from(sections.values());
-        const fileName = textEditor.document.fileName;
+        const res = this.calculateEditorData(textEditor);
+        if (res === undefined) { return ; }
```

Comment:
minor code styling, could we use `if (!res) { return ; }` instead to better match the rest of the file?

Reply:
done